packages/react-router-dom/modules/BrowserPrompt.js

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react'
+import { Prompt } from 'react-router'
+
+const BEFOREUNLOAD = "beforeunload"; // see: http://stackoverflow.com/questions/39094138/reactjs-event-listener-beforeunload-added-but-not-removed
```

Comment:
I'm not sure what this is for. The stackoverflow question has to do with the function being the same, not the string `"beforeunload"`.